bats/tests/snapshots/create-use-snapshot.bats

```diff
@@ -90,7 +92,14 @@ local_setup() {
     assert_success
     refute_output ""
     snapshot_id=$output
```

Comment:
`snapshot_id` should be renamed to `snapshot_name`.

Or maybe use `$SNAPSHOT` again:

```
SNAPSHOT=$output
```

because mixing `SNAPSHOT` and `snapshot_name` looks weird.

In that case I would also rename `snapshot_description` to `DESCRIPTION`.

Reply:
`SNAPSHOT` is global in this file. Those lower-case names are local to a test or function.

And I deliberately used the name `snapshot_id` to show that the name I'm assigning is an existing ID.